edxval/views.py

```diff
@@ -268,10 +268,10 @@ class HLSMissingVideoView(APIView):
     """
     authentication_classes = (OAuth2Authentication, SessionAuthentication)
 
-    def get(self, request):
-        courses = request.query_params.getlist('courses')
-        batch_size = int(request.query_params.get('batch_size', 50))
-        offset = int(request.query_params.get('offset', 0))
+    def post(self, request):
+        courses = request.data.get('courses')
+        batch_size = int(request.data.get('batch_size', 50))
+        offset = int(request.data.get('offset', 0))
```

Comment:
why we need to convert these to `int`, shouldn't they already be in integer format?